cloudcv17/settings.py

```diff
@@ -20,7 +20,7 @@
 # See https://docs.djangoproject.com/en/1.6/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '9%$in^gpdaig@v3or_to&_z(=n)3)$f1mr3hf9e#kespy2ajlo'
+SECRET_KEY = config.SECRET_KEY
```

Comment:
Instead of declaring again, lets make an import at the start of the file like `from config import *`